bump/action.yml

```diff
@@ -67,8 +67,7 @@ outputs:
       (or none were found matching the version-prefix, if provided)
   next-version:
```

Comment:
We can also decide to add a new output parameter for this change (making this PR less breaking), but we still have modified behavior for error-cases w/ GH interactions during SemVer bump strategy (which now enforces the output to be `""`  instead of the intended version).

Reply:
I would not be opposed to keeping `current-version` and `next-version` as strings and defining a new output that more accurately describes the JSON object (`next-version-json`, `bump-info`, `bump-metadata`, ...).

Reply:
Please refer to b21edd8ec6f98f8d86548a758dc2a15090d002b9 for the introduction of `next-version-json` next to the current `next-version` output parameter.